tests/sync/session.cpp

```diff
@@ -64,6 +79,20 @@ std::shared_ptr<SyncSession> sync_session(SyncServer& server, std::shared_ptr<Sy
 
 namespace {
 
+void add_objects(Realm::Config& config, int count=2)
```

Comment:
I have some vague notions of tests that might be able to use them, but I think it makes sense to inline this code and only break it out if we do end up needing test objects for other tests.